benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -316,11 +318,37 @@ def get_all_rearrange_operators():
 
     # Filter for Array Rearrange operators
     rearrange_mx_operators = {}
-    for op_name, op_params in mx_operators.items():
+    for op_name, _ in mx_operators.items():
         if op_name in rearrange_ops and op_name not in unique_ops:
             rearrange_mx_operators[op_name] = mx_operators[op_name]
     return rearrange_mx_operators
 
+
+def get_all_indexing_routines():
+    """Gets all indexing routines registered with MXNet.
+
+    Returns
+    -------
+    {"operator_name": {"has_backward", "nd_op_handle", "params"}}
+    """
+    # @ChaiBapchya unravel_index errors out on certain inputs
```

Comment:
is this a TODO ?

Reply:
It is an open issue whose link I have already added.